darglint2/config.py

```diff
@@ -78,27 +79,50 @@ def from_string(cls, level: str) -> "LogLevel":
             raise ValueError('Unrecognized log level, "{}"'.format(level))
 
 
-class Configuration(object):
+class Configuration:
+    """
+    A dataclass representing a configuration.
```

Comment:
Is it confusing to call this a "dataclass" when a concept with that name exists in the Python standard library, but this class doesn't actually use it?

Reply:
Good question

Reply:
It kind of is. Do you have an idea for a different phrasing?